src/Analyzer.JsonRuleEngine/Expressions/Expression.cs

```diff
@@ -42,10 +42,10 @@ internal Expression(ExpressionCommonProperties commonProperties)
         /// Executes this <see cref="Expression"/> against a template.
         /// </summary>
         /// <param name="jsonScope">The specific scope to evaluate.</param>
-        /// <param name="jsonLineNumberResolver">An <see cref="ILineNumberResolver"/> to
+        /// <param name="jsonLineNumberResolver">An <see cref="ILocationResolver"/> to
```

Comment:
and while updating the comments, also update the Readmes (not sure if it is there). 
And the Readme should say there is Bicep too in the tool.

Reply:
READMEs are updated